environment.yml

```diff
@@ -29,7 +29,6 @@
 # aimed at NEST developers. You can specify a different name by passing
 # -n my_name to the conda env create command.
 
-name: nest-simulator-dev
```

Comment:
I don't know much about conda, but doesn't removing the `-` lead to invalid YAML somehow?

Reply:
The `-` is from diff. Conda expects a dictionary on the top level with keys `name`, `channels`, `dependencies`, etc. Giving a name in the `environment.yml` is however the old implicit named environment syntax. There are various reasons why environments should be by path (i.e. `conda env create -p xyz` instead of `conda env create -n xyz`), and removing this from the environment.yml will make the user explicitly aware. Both, name and path, are perfectly possible, but the user has to make a conscious choice. Giving the recommended way `-p …` in the docs then follows conda's best practice of prefering path over name and still leaves the user with a working copy-paste solution.

Reply:
Thanks for your explanation!